src/builtin.rs

```diff
@@ -46,6 +48,45 @@ pub static MISSING_GLOBALS: &'static [&'static str] = &[
     "distribs",
 ];
 
+static HTML_CONVERTER: Lazy<htmd::HtmlToMarkdown> = Lazy::new(|| {
```

Comment:
Since rust 1.80.0, std has had [`LazyLock`](https://doc.rust-lang.org/std/sync/struct.LazyLock.html), which does pretty much the same thing. Not adding another third-party dependency would be nice.

Reply:
Good point, changed. Thanks!